mimecast/plugin.spec.yaml

```
@@ -1,21 +1,25 @@
 plugin_spec_version: v2
 extension: plugin
-product: ["insightconnect"]
+products: ["insightconnect"]
 name: mimecast
 title: Mimecast
 description: Services for email security, archiving and continuity. Protect, manage
   and archive without compromise
-version: 3.0.0
+version: 3.0.1
```

```suggestion
version: 3.1.0
```
New action = minor version bump